src/changelog/cli/_posts/2023-02-06-1-28-0.md

```diff
@@ -0,0 +1,15 @@
+---
+modified_at: 2023-02-06 16:00:00
```

Comment:
Don't forget to update the date here and in the filename before merging so that it matches the date of release